ggv_generator.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import pandas as pd
+import itertools
+import engine
+import multiprocessing
+from tqdm import tqdm
+from time import perf_counter
+
+def ggv_generator(vehicle_params, sweep_ranges:dict, mesh:int):
```

Comment:
Instead of deleting the old solver class, and just replacing it with the GGV generator, i think it's better if there's a generic class that handles the sweeping of points. The GGV should be an output of the sweep (maybe a method within a generic solver class), not its own class

Reply:
The GGV isnt a class, its a method. GGV is needed for lapsim, and MMM is needed for MMM. Thats why I made them separate, however you may be right on adding another sweeping function in a different file